addons/goggles/functions/fnc_externalCamera.sqf

```diff
@@ -6,15 +6,13 @@
  * None
  *
  * Return Value:
- * Whether the camera is in external view or not. If the "showInThirdPerson" option is checked, this will always return false. <BOOL>
+ * Whether the camera is in external view or not. <BOOL>
  *
  * Example:
- * call ace_goggles_fnc_removeRainEffect;
+ * call ace_goggles_fnc_externalCamera;
  *
  * Public: Yes
  */
 #include "script_component.hpp"
 
-if (GVAR(showInThirdPerson)) exitWith { false };
-
-(cameraView in ["EXTERNAL", "GROUP"] || {call EFUNC(common,isFeatureCameraActive)})
+cameraView in ["EXTERNAL", "GROUP"] || EFUNC(common,isFeatureCameraActive) // return
```

Comment:
The implemenation of lazy evaluation is nothing more than the addition of the alternative syntax of `&&` and `||` which is `BOOL && CODE` etc.